docs/concepts/worklist.md

```diff
@@ -0,0 +1,140 @@
+﻿# Worklist (UPS-RS) Support
+
+The DICOM service supports a subset of the Worklist Service (UPS-RS) defined in the DICOMweb&trade; Standard. Partially supported transactions include:
+
+- [Create](#create)
+- [Search](#search)
+
+The base URI for all operations below should include the [desired API version](../api-versioning.md) and the [data partition](data-partitions.md) if that feature is enabled.
+Throughout, the variable  in a URI template stands for a Workitem UID.
+
+## Create
+
+This transaction uses the POST method to create a new Workitem.
+
+| Method | Path               | Description |
+| :----- | :----------------- | :---------- |
+| POST   | `../workitems`         | Create a Workitem. |
+| POST   | `../workitems?{workitem}` | Creates a Workitem with the specified UID. |
+
+
+If not specified in the URI, the payload dataset must contain the Workitem SOPInstanceUid.
```

Comment:
Should we say about AffectedSOPIntanceUID. Since we will be also looking into that